tokio/src/runtime/task/list.rs

```diff
@@ -140,34 +138,36 @@ impl<S: 'static> OwnedTasks<S> {
 
     /// Shuts down all tasks in the collection. This call also closes the
     /// collection, preventing new items from being added.
-    pub(crate) fn close_and_shutdown_all(&self)
+    ///
+    /// The parameter start should be random among different worker threads
+    /// to reduce lock conflicts during shutdown.
+    /// Initiate shutting down the shard indexed by the start, and reset to 0
+    /// once the shar_size is reached, continuing until start - 1, it works like a ring.
```

Comment:
This sentence is inaccurate if `start == 0`. 
```suggestion
    /// once the `shard_size` is reached, continuing until `start - 1`, it works like a ring.
```